File: backend/src/main/java/com/example/HopitalPlanningProject/model/Equipe.java
package com.example.HopitalPlanningProject.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Equipe {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idEquipe;

    private String nomEquipe;

    @OneToOne
    @JoinColumn(name = "idPlanning", unique = true, nullable = false)
    private Planning planning;
}

File: frontend/src/models/Equipe.js
// models/Equipe.js
class Equipe {
    constructor(idEquipe, nomEquipe, planning) {
        this.idEquipe = idEquipe;
        this.nomEquipe = nomEquipe;
        this.planning = planning; // Planning est un objet ou ID
    }
}

export default Equipe;
